cdn/.version

```diff
@@ -1 +1 @@
-9.3.0
+${npm_package_version} 
```

Comment:
After you bump the version in the package.json you'll want to run `yarn build` again which should fix this. If not, you can just manually set this file to the same value as the `version` in the packages.json.

Reply:
I bumped the version in package.json, however, I saw this file has been deleted in the `master` branch.

let me known if I have anything else to do on my side

Reply:
Nope, nothing else to do here, your good 👍